src/app/components/Footer/index.test.jsx

```diff
@@ -0,0 +1,7 @@
+import Footer from './index';
+
+import testHelper from '../../../../src/__test__/testHelper';
+
+describe(`Footer`, () => {
+  testHelper.shouldMatchSnapshot('Footer', Footer);
```

Comment:
We should use the snapshot test helper that is on latest: https://github.com/bbc/simorgh/blob/latest/src/__test__/snapshotTestHelper.js